### PR TITLE
Add UT for session module

### PR DESCRIPTION
**Pull request by 2b3c511**

### CODE REVIEW

File: iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    sessionPool.insertRecordsOfOneDevice(
+        "device1", timeList, measurementsList, typesList, valuesList);
+  }
+
+  @Test
+  public void testInsertRecords2() throws Exception {
+    ConcurrentLinkedDeque<ISession> queue = new ConcurrentLinkedDeque<>();
+    queue.add(session);
+    Whitebox.setInternalState(sessionPool, "queue", queue);
+    // 调用 insertRecords 方法
```
Comment (Beyyes) [inline iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java:251]: remove chinese

File: iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+      sessionPool.close();
+    }
+  }
+
+  @Test(expected = IoTDBConnectionException.class)
+  public void testInsertRecords() throws Exception {
+    Mockito.doThrow(new IoTDBConnectionException(""))
+        .when(session)
+        .insertRecords(anyList(), anyList(), anyList(), anyList(), anyList());
+    // 调用 insertRecords 方法
```
Comment (Beyyes) [inline iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java:90]: remove chinese

File: iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  @Mock private Session session;
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+
+    sessionPool = new SessionPool(Arrays.asList("host:11"), "user", "password", 10);
+    ConcurrentLinkedDeque<ISession> queue = new ConcurrentLinkedDeque<>();
+    queue.add(session);
+    // 设置 SessionPool 对象的内部状态
```
Comment (Beyyes) [inline iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java:73]: chinese

File: iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java (hunk 2 above)
Comment (2b3c511) [reply iotdb-client/session/src/test/java/org/apache/iotdb/session/pool/SessionPoolExceptionTest.java]: done

**Pull request merged**